mod-consortia/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@azizbekxm please update teams-assignment file in the root folder as well

Reply:
Updated